setup.cfg

```diff
@@ -3,7 +3,7 @@ zip_safe = False
 include_package_data = True
 packages = find:
 install_requires =
-    dffml @ https://github.com/intel/dffml/archive/058d0b300c5582c4ee773ef29971ceda547e36e1.zip
+    dffml @ https://github.com/intel/dffml/archive/fe0fe1268a9dac07f6d63bef049f4af032b04157.zip
```

Comment:
This file is a manifest: https://github.com/intel/dffml/blob/main/docs/arch/0008-Manifest.md